verilog/analysis/verilog_linter.cc

```diff
@@ -59,6 +59,8 @@ ABSL_FLAG(std::string, rules_config, ".rules.verible_lint",
           "Path to lint rules configuration file.");
 ABSL_FLAG(verilog::RuleSet, ruleset, verilog::RuleSet::kDefault,
           "[default|all|none], the base set of rules used by linter");
+ABSL_FLAG(std::string, waivers_file, "",
+          "Path to waivers file.");
```

Comment:
Point to some description of waiver file format.
Or add a `--help_waivers_file` flag to print that description and exit.

Reply:
Added some documentation to readme and expanded the help messages to point to it.